packages/asset-uploader/src/env.ts

```diff
@@ -0,0 +1,3 @@
+export const env = {
+  VERCEL_ENV: process.env.VERCEL_ENV,
```

Comment:
can we please abstract away VERCEL_ENV and use something that will not change no matter if we use vercel or something else?